js/ui/viewSelector.js

```diff
@@ -453,6 +453,99 @@ const ViewsDisplay = new Lang.Class({
     }
 });
 
+const ViewsClone = new Lang.Class({
+    Name: 'ViewsClone',
+    Extends: St.Widget,
+
+    _init: function(viewSelector, viewsDisplay, forOverview) {
+        let settings = Clutter.Settings.get_default();
+
+        this._viewSelector = viewSelector;
+        this._viewsDisplay = viewsDisplay;
+        this._forOverview = forOverview;
+
+        let appDisplay = this._viewsDisplay.appDisplay;
+        let entry = new ShellEntry.OverviewEntry();
+        entry.reactive = false;
+        entry.clutter_text.reactive = false;
+
+        let iconGridClone = new Clutter.Clone({ source: appDisplay.gridActor,
+                                                x_expand: true,
+                                                y_expand: true });
+
+        let appGridContainer = new AppDisplay.AllViewContainer(iconGridClone);
+        appGridContainer.reactive = false;
+
+        let layoutManager = new ViewsDisplayLayout(entry, appGridContainer, null);
```

Comment:
Technically, the second parameter of ViewsDisplayLayout's constructor is an `appDisplay` which is no longer true when creating the clone (it's a `AllViewContainer`) so I think I'll rename some names in ViewsDisplayContainer and ViewsDisplayLayout to make it more generic (e.g. maybe call it 'grid container')